jenkins/jobs/trigger/jenkins_job_trigger.py

```diff
@@ -59,7 +68,8 @@ def main():
                   'VC_BUILD': args.vc_build,
                   'BUILD_NUM': args.build_num}
         jenkinsci.build_job(jobname=args.job_name, params=params)
-        LOG.debug('Triggered job %s with parameters %s.', args.job_name, params)
+        LOG.debug('Triggered job %s with parameters %s.', args.job_name,
```

Comment:
I actually dislike this extremely short line wrapping but if it's not configurable then I prefer linting.
(80 chars made sense when you're dealing with punch cards but these days I prefer one logical statement per line and people can configure their IDE to wrap at a boundary if it bothers them).

Reply:
For better or worse, there is a strong python convention here: https://www.python.org/dev/peps/pep-0008/#id19

I too prefer longer lines, but I think sticking with the standard style guide would be best.